src/cargo/util/toml/mod.rs

```diff
@@ -763,6 +764,7 @@ impl TomlManifest {
             deps,
             me.features.clone().unwrap_or_else(BTreeMap::new),
             project.links.clone(),
+            project.namespaced_features.unwrap_or(false),
```

Comment:
Ah I just remembered, can this be gated behind a feature as well? (setting this key at all)

Reply:
Sure - not sure what kind of gate you had in mind?

Reply:
Oh something like `cargo-features = ["namespaced-features"]` in `Cargo.toml`, you can see some other examples of this via `src/cargo/core/features.rs` I think

Reply:
For now yeah we'll want that (the Cargo feature basically unlocks the `namespaced-features` config value).

Long term we may not have both but for the near-term I think we'll want that.